datacite-commons/src/main/java/no/unit/nva/doi/DoiClient.java

```diff
@@ -2,14 +2,13 @@
 
 import java.net.URI;
 import no.unit.nva.model.Publication;
-import no.unit.nva.publication.model.business.Resource;
```

Comment:
This file contains the meaningful change, which in many ways indicates that we did at least one thing right :D